py/private/venv/venv.bzl

```diff
@@ -82,6 +82,13 @@ def _make_venv(ctx, name = None, main = None, strip_pth_workspace_root = None):
             "map_each": _pth_import_line_map,
         })
 
+    # A few imports rely on being able to reference the root of the runfiles tree as a Python module,
+    # the common case here being the @rules_python//python/runfiles target that adds the runfiles helper,
+    # which ends up in bazel_tools/tools/python/runfiles/runfiles.py, but there are no imports attrs that hint we
+    # should be adding the root to the PYTHONPATH
+    # Maybe in the future we can opt out of this?
```

Comment:
I'm not sure I follow the suggestion? How would you use the runfiles helper to import the runfiles helper?

Reply:
oh I see. In rules_nodejs we just exposed an environment variable to all actions to help them find the runfiles helper.

If the python runfiles library were published to pypi, and fetched as a pip dep, could it naturally work that way?

Reply:
You'd still need the root of the runfiles tree as a package, as the import path that's documented (and therefore likely in everyones code) starts with `bazel_tools`, unless rules_py special cased that dependency and nested it somehow.

Yeah, if this were a regular pypi package it would fall though the normal paths.

Reply:
Alternativy, rules_py vendors the runfiles helper and exposes it somewhere else at a nested package or something. Would mean a breaking change for users.
Also I wonder how many things rely on this behaviour.

Reply:
so you could publish a `bazel_tools` package to pypi and users imports would continue to work, right? seems like a more principled answer, but I don't mean to block a merge here